shepherd_bms/src/compute.cpp

```diff
@@ -61,7 +61,7 @@ FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint1
     return NOT_FAULTED;
 }
 
-bool ComputeInterface::isCharging() // This is useless kinda, especially if we move to DCDC
+bool ComputeInterface::isChargingEnabled() // This is useless kinda, especially if we move to DCDC
```

Comment:
being a bit picky but i think calling it isChargerConnected is more descriptive of what this really means than isChargingEnabled